src/lib/AnnotationControls.ts

```diff
@@ -153,6 +154,16 @@ export default class AnnotationControls {
         this.resetControls();
         this.currentMode = mode;
         this.updateButton(mode);
+
+        this.isModeTemp = mode !== AnnotationMode.NONE;
```

Comment:
It seems like this should use a separate setter that accepts a boolean and is not based on `mode` at all.